website/contributing/contributing-intro.md

```diff
@@ -0,0 +1,130 @@
+
+## Ways to Contribute
+
+If you are eager to start contributing code right away, we have a list of [good first issues](https://github.com/facebook/react-native/labels/good%20first%20issue) that contain bugs which have a relatively limited scope. 
+As you gain more experience and demonstrate a commitment to evolving React Native, you may be granted issue management permissions in the repository.
+
+There are other ways you can contribute without writing a single line of code. Here are a few things you can do to help out:
+
+1. **Replying and handling open issues.** 
+
+   We get a lot of issues every day, and some of them may lack necessary information. You can help out by guiding people through the process of filling out the issue template, asking for clarifying information, or pointing them to existing issues that match their description of the problem. 
+   We cover more about this process on the [Triaging GitHub Issues](/contributing/triaging-github-issues) page.
+
+2. **Reviewing pull requests for the docs.** 
+
+   Reviewing [documentation updates](https://github.com/facebook/react-native-website/pulls) can be as simple as checking for spelling and grammar. 
+   If you encounter situations that can be explained better in the docs, click **Edit** at the top of most docs pages to get started with your own contribution.
+
+3. **Help people write test plans.** 
+
+   Some pull requests sent to the main repository may lack a proper test plan. These help reviewers understand how the change was tested, and can speed up the time it takes for a contribution to be accepted.
+
+Each of these tasks is highly impactful, and maintainers will greatly appreciate your help.
+
+### Our Development Process
+
+We use GitHub issues and pull requests to keep track of bug reports and contributions from the community. All changes from engineers at Meta will sync to [GitHub](https://github.com/facebook/react-native) through a bridge with Meta's internal source control. Changes from the community are handled through GitHub pull requests. 
+
+Once a change made on GitHub is approved, it will first be imported into Facebook's internal source control and tested against Facebook's codebase. Once merged at Facebook, the change will eventually sync back to GitHub as a single commit once it has passed Facebook's internal tests.
+
+You can learn more about the contribution process in the following documents:
+
+- [Triaging GitHub Issues](/contributing/triaging-github-issues)
+- [Managing Pull Requests](/contributing/managing-pull-requests)
+
+We also have a thriving community of contributors who would be happy to help you get set up. You can reach out to us through [@ReactNative](http://twitter.com/reactnative) (the React Native team) and [@ReactNativeComm](http://twitter.com/reactnativecomm) (the React Native Community organization).
```

Comment:
```suggestion
We also have a thriving community of contributors who would be happy to help you get set up. You can reach out to the React Native team through [@ReactNative](http://twitter.com/reactnative).
```

The react native community twitter account has been very inactive for quite some time

Reply:
Oh, let's remove that mention for now.